app/scripts/modules/netflix/chaosMonkey/chaosMonkeyExceptions.directive.js

```diff
@@ -36,7 +36,9 @@ module.exports = angular
           this.accounts = details;
           this.regionsByAccount = {};
           details.forEach((account) => {
-            this.regionsByAccount[account.name] = ['*'].concat(account.regions.map((region) => region.name));
+            if (account.regions) {
+              this.regionsByAccount[account.name] = ['*'].concat(account.regions.map((region) => region.name));
+            }
```

Comment:
Might want to restrict the accounts to `aws` ones - I don't think Titus is supported by Chaos Monkey (at least not yet).